lib/editor/atto/plugins/kalturamedia/yui/build/moodle-atto_kalturamedia-button/moodle-atto_kalturamedia-button-debug.js

```diff
@@ -53,43 +54,56 @@ Y.namespace('M.atto_kalturamedia').Button = Y.Base.create('button', Y.M.editor_a
                 callback: this._kalturamedia
             });
     },
-    _kalturamedia: function(){
+    _kalturamedia: function() {
         this._currentSelection = this.get('host').getSelection();
         if (this._currentSelection === false) {
             return;
         }
+
+        var embedButton = "<button id='KalturaMediaSubmit' disabled='disabled' hidden='hidden' style='display: none'>" + M.util.get_string('embedbuttontext', COMPONENTNAME) + "</embed>";
+
+        var height = 580;
+        var width = 1100;
+        var panelHeight = height + 90;
+        var panelWidth = width + 23 + 'px';
+        width += 'px';
+
+        if (Y.UA.ipod !== 0 || Y.UA.ipad !== 0 || Y.UA.iphone !== 0 || Y.UA.android !== 0 || Y.UA.mobile !== null) {
```

Comment:
would bunch to a function that get enum (mobile/ios/android) and returnsa true/false